src/protocol/packet/server.rs

```diff
@@ -346,6 +346,12 @@ pub struct SSpawnMe {
     pub is_lord: bool, // TODO try to identify the usage of the field
 }
 
+#[derive(Clone, Deserialize, Serialize, PartialEq, Debug)]
+pub struct SReturnToLobby {}
+
+#[derive(Clone, Deserialize, Serialize, PartialEq, Debug)]
+pub struct SExit {}
```

Comment:
S_EXIT has two fields. Two i32 that are returned to the launcher. Please include them and set the values that the client normal would.

Reply:
Testdata for S_EXIT (normal logout):
[ 0x0, 0x0, 0x0, 0x0, 0x10, 0x0, 0x0, 0x0 ]